sdks/python/apache_beam/transforms/external.py

```diff
@@ -180,14 +180,52 @@ def _get_named_tuple_instance(self):
 
 
 class SchemaTransformPayloadBuilder(PayloadBuilder):
-  def __init__(self, identifier, **kwargs):
-    self._identifier = identifier
+  def __init__(self, schematransform_config, strict_schema=False, **kwargs):
+    self._schematransform_config = schematransform_config
+    self._strict_schema = strict_schema
     self._kwargs = kwargs
 
+  def _get_schema_proto_and_payload(self, **kwargs):
```

Comment:
do you also want to check that there are no kwargs beyond those in the schema?

Reply:
right, `strict_schema` flag should catch that too. will add that check

Reply:
yeah good idea, will add that check